build.gradle

```diff
@@ -20,6 +19,7 @@ plugins {
 allprojects {
     repositories {
         google()
+        mavenLocal()
```

Comment:
should we just keep this here all the time for r-b?

Reply:
Probably not. :)

Reply:
why not?

Reply:
Mistakenly using locally build artifacts in error would be more common and lead to side-effects without realizing. (e.g You may have a local artefact that's much older than one published but newly added component uses the published one).

Being explicit is probably better.

Reply:
Right: definitely do this.  Upstream repositories are "the same" for everybody; `mavenLocal()` sure isn't the same for everybody.